pkg/scheduler/admission/podnodeconstraints/api/types.go

```diff
@@ -0,0 +1,14 @@
+package api
+
+import (
+	"k8s.io/kubernetes/pkg/api/unversioned"
+)
+
+// Config is the configuration for the pod node name and node selector
+// constraint plug-in. It contains two booleans to allow or prohibit
+// the use of nodeName and nodeSelector fields in pod requests.
+type Config struct {
+	unversioned.TypeMeta
+	ProhibitNodeName     bool
```

Comment:
You should add the same godoc comments that you have in the v1 version of the type here (just omit the struct tags)